src/internal/operators/bufferTime.ts

```diff
@@ -209,9 +219,7 @@ class BufferTimeSubscriber<T> extends Subscriber<T> {
   }
 
   closeContext(context: Context<T>) {
-    this.destination.next(context.buffer);
     const contexts = this.contexts;
```

Comment:
No offense, but I've never seen this pattern before. Is there a dogma with the `this` keyword in this project or something? This aliasing seems to be done more out of superstition than logic, and makes the code a bit inscrutable.  I tried to follow existing patterns, but some of them make no sense, and I would like to discuss changing things in future PRs as I cut my teeth w/ this code base.